docs/upgrade_with_cli.md

```diff
@@ -44,17 +44,112 @@ During the upgrade of Percona Everest, only Everest and Everest operator are upg
         2024-05-03T12:07:28Z    info    upgrade/upgrade.go:148  Everest has been upgraded to version 0.10.0 {"component": "upgrade"}
         ```
 
+3. After upgrading, refresh the Percona Everest UI to access the new version.
 
 
+### Upgrading to Percona Everest 1.2.0
 
-3. After upgrading, refresh the Percona Everest UI to access the new version.
+When upgrading to 1.2.0 using the CLI command `everestctl upgrade`, all your existing backup storages and monitoring instances will be automatically migrated to the namespaces specified in their `.spec.allowedNamespaces` fields.
+
+Following the upgrade, your databases should not experience any downtime. Your backup, restore, and monitoring functionalities should continue to operate normally. 
 
+In the unlikely event that your upgrade fails, and you need to manually migrate these resources, follow the steps in [how to resolve upgrade failures in Percona Everest 1.2.0](#how-to-resolve-upgrade-failures-in-percona-everest-120) section.
 
 
 ## How to address a failed upgrade
 
 If the upgrade fails, you can attempt it again. If the issue persists, [create a GitHub issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-a-repository).
 
+### How to resolve upgrade failures in Percona Everest 1.2.0
+
+In the unlikely event that your upgrade fails and you need to manually migrate these resources, follow the steps below:
+{.power-number}
+
```

Comment:
Since this section is separate from the section about the breaking API changes, I feel that we must at least add a note about the breaking changes and link to the document.

```suggestion
### Resolving upgrade failures due to the breaking API changes in 1.2.0

Everest 1.2.0 includes some breaking API changes (attach link to doc). While all your resources will be migrated automatically, in the unlikely event that your upgrade fails and you need to manually migrate these resources, follow the steps below:
{.power-number}

```